packages/typespec-python/test/azure/generated/payload-multipart/payload/multipart/_vendor.py

```diff
@@ -19,16 +19,19 @@
     from typing import MutableMapping  # type: ignore  # pylint: disable=ungrouped-imports
 
 
-class NamedBytesIO(BytesIO):
-    def __init__(self, name: str, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.name = name
+JSON = MutableMapping[str, Any]  # pylint: disable=unsubscriptable-object
 
+# file-like tuple could be `(filename, IO (or bytes))` or `(filename, IO (or bytes), content_type)`
+FileType = Union[IOBase, bytes]
```

Comment:
I would use the type hints I have here from core https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/core/azure-core/azure/core/rest/_helpers.py#L79